src/ResourceManager/Automation/Commands.Automation/help/Remove-AzureRMAutomationSchedule.md

```diff
@@ -29,12 +29,12 @@ The **Remove-AzureRmAutomationSchedule** cmdlet deletes a schedule from Azure Au
 PS C:\>Remove-AzureRmAutomationSchedule -AutomationAccountName "Contoso17" -Name "Schedule01" -ResourceGroupName "ResourceGroup01"
```

Comment:
Can you add a line to the changelog for Automation that mentions this change (https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Automation/Commands.Automation/ChangeLog.md)

Reply:
Done!